play/src/front/Components/Cowebsites/JistiCowebsiteComponent.svelte

```diff
@@ -0,0 +1,30 @@
+<script lang="ts">
+    import { onMount } from "svelte";
+    import { JitsiCoWebsite } from "../../WebRtc/CoWebsite/JitsiCoWebsite";
+    import { jitsiExternalApiFactory } from "../../WebRtc/JitsiExternalApiFactory";
+
+    export let actualCowebsite: JitsiCoWebsite;
+    let options = actualCowebsite.getOptions();
+    let domain = actualCowebsite.getDomain();
+    let jitsiContainer: HTMLDivElement;
+
+    onMount(() => {
+        jitsiExternalApiFactory
+            .loadJitsiScript(domain)
+            .then(() => {
+                options.parentNode = jitsiContainer;
+                new window.JitsiMeetExternalAPI(domain, options);
+            })
+            .catch((e) => {
+                console.error(e);
+            });
+    });
+</script>
+
+<div bind:this={jitsiContainer} class="w-full height" />
+
+<style>
+    .height {
+        height: 90%;
+    }
```

Comment:
90% will not be perfect for all screen heights.
Instead, try to put the tabs bar and the content (cowebsite/Jitsi) in a common flex div.
The goal: the cowebsite / Jitsi div should be set to 100% height without overflowing from the viewport